### ISSUE TITLE
session sharing works in read-only mode

### ISSUE THREAD

**Issue by medfki** [cedrozor/myrtille#183]
Hello,
First of all, thank you very much for this excellent project.
I noticed two problems related to session sharing in the new version of Myrtille 2.7.0
1- the session does not appear systematically after the execution of the shared link, it is necessary to move the mouse in the main session so that the zones appear in the shared window.

2- the shared session is still in read-only mode (preview only)
the "Grant control" check box in the "shared session" floating window does not affect link creation

which is not the case for version 2.5.5

**Comment by cedrozor** [comment]
Hi,

Thanks for the report. It looks like a refresh issue and a regression for session sharing, I will investigate.

**Comment by cedrozor** [comment]
I found the problem! :) it was a regression after the improvement of memory usage in the 2.7.0 version: I will hotfix it tomorrow, or you can add the following at line 547 into RemoteSessionManager.cs:
```
if (!_guestsIdleTimeout.ContainsKey(session.SessionID))
{
  _guestsIdleTimeout.Add(session.SessionID, new CancellationTokenSource());
}
```

**Comment by cedrozor** [comment]
Fixed into 2.7.0 (updated release, you can re-download/re-install it)
